### PR TITLE
Clarify install commands in README

### PR DESCRIPTION
**Pull request by rd4398**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,14 +9,16 @@ install that simulates Ironic multi-tenancy through the use of `project_owner_id
...
-To  install the python code:
+To  install the python code without PyPI:
 
 ```
     $ git clone https://github.com/CCI-MOC/esi-leap
     $ cd esi-leap
     $ sudo python setup.py install
 ```
 
+To install the package using PyPI:
```
Comment (tzumainn) [inline README.md:20]: I'd put this before the instructions for installing from source; it's what we want people to do. Also, maybe reword as "To install as a package"

File: README.md [hunk 2]
```diff
@@ -9,14 +9,16 @@ install that simulates Ironic multi-tenancy through the use of `project_owner_id
 
 ### Installation
 
-To  install the python code:
+To  install the python code without PyPI:
```
Comment (tzumainn) [inline README.md:12]: "To install from source"

**Review by tzumainn — CHANGES_REQUESTED** [review]
A few minor comments!

**Comment by rd4398** [comment]
Made changes as per your feedback!
Let me know if anything else needs to be changed.
